### PR TITLE
Refine doc of PDS dataset allocation

### PR DESCRIPTION
**Pull request by std4lqi**
Signed-off-by: Qi Liang <31643673+std4lqi@users.noreply.github.com>

Update for #115

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]
LGTM!
Thanks for updating the system tests
Will merge once the merge conflicts are resolved 😋

**Review by tiantn — APPROVED** [review]

**Review by lauren-li — APPROVED** [review]

**Pull request merged**
